lib/info.js

```diff
@@ -13,10 +13,14 @@ const getInfo = async (dhpkgid) => {
 
   const firstParagraphReadme = `# ABOUT\n${readme.substring(0, 200)}... *see more below*`
 
-  let resourcesInfo = '\n# RESOURCES\n'
+  let resourcesInfo = `
+  \n# RESOURCES\n
+  | Name | Format |
+  |------|--------|
+  `
 
   metadata.descriptor.resources.forEach(resource => {
-    resourcesInfo = `${resourcesInfo}* **${resource.name}** [${resource.format}]\n`
+    resourcesInfo += `|${resource.name}|${resource.format}|\n`
```

Comment:
Only thing here is that if resource does not have format it displays undefined
![resources](https://user-images.githubusercontent.com/15143785/26961544-91647fc8-4cf0-11e7-825e-affe14af4726.png)

`resource.format || ''N/A'` would fix this